### PR TITLE
Zuber8040 patch 1

### PR DESCRIPTION
**Pull request by Zuber8040**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4254"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/Zuber8040.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"type": "personal",
+	"bio": "JavScript Developer || React Js|| Tailwind|| Open Source Contributor  ",
+	"links": [{
+			"name": "follow on Twitter",
+			"url": "https://twitter.com/Zuber763",
+			"icon": "FaTwitter"
+		},
+		{
+			"name": "Connect onLinkedin ",
+			"url": "https://www.instagram.com/eddieandsaraexplore/",
```
Comment (kumarsonsoff3) [inline data/Zuber8040.json:13]: Please update it with your LinkedIn URL. 🛠

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by Zuber8040 — COMMENTED** [review]
I have added the linkdln link

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**